docs/applications/node.md

```diff
@@ -3,11 +3,141 @@ id: node
 title: Light Client
 ---
 
-Bitcoin-S comes bundled with a light client Bitcoin node. this client
-is capable of doing verification of some parts of the blockchain,
-and can act as a starting point for building Bitcoin applications
-that need to connect to the P2P network.
-
-This node is currently only released as a library, and not as a binary.
-This is because it (nor the documentation) is not deemed production
-ready. Use at your own risk, and without too much money depending on it.
+Bitcoin-s has node module that allows you to connect to the p2p network.
+
+### Neutrino Node
+
+Bitcoin-s has experimental support for neutrino which is a new lite client proposal on the bitcoin p2p network. You can
+read more about how neutrino works [here](https://suredbits.com/neutrino-what-is-it-and-why-we-need-it/).
+
+#### Callbacks
+
+We support call backs for the following events that happen on the bitcoin p2p network:
+
+1. onTxReceived
+2. onBlockReceived
+3. onMerkleBlockReceived
+4. onCompactFilterReceived
+
+That means every time one of these events happens on the p2p network, we will call your callback
+so that you can be notified of the event. Let's make a easy one
+
+#### Example
+
+Here is an example of constructing a neutrino node and registering a callback so you can be notified of an event
+
+```scala mdoc:invisible
+import akka.actor.ActorSystem
+import org.bitcoins.core.protocol.blockchain.Block
+import org.bitcoins.node._
+import org.bitcoins.testkit.node._
+import org.bitcoins.testkit.node.fixture._
+import org.bitcoins.testkit.rpc._
+import org.bitcoins.server.BitcoinSAppConfig
+import org.bitcoins.testkit._
+import org.bitcoins.testkit.chain._
+import scala.concurrent._
+import scala.concurrent.duration._
+import java.nio.file.Files
+import com.typesafe.config.ConfigFactory
+```
+
+```scala mdoc:compile-only
+
+implicit val system = ActorSystem(s"node-example")
+implicit val ec = system.dispatcher
+
+//we also require a bitcoind instance to connect to
+//so let's start one
+val instance = BitcoindRpcTestUtil.instance()
+val p2pPort = instance.p2pPort
+val bitcoindF = BitcoindRpcTestUtil.startedBitcoindRpcClient(instance)
```

Comment:
this isn't the right version of bitcoind, we need to make sure we get one that supports neutrino.